packages/common/src/index.ts

```diff
@@ -1,4 +1,6 @@
 import useSWR from "swr";
+import { useParams } from "react-router";
+import { isAddress } from "viem";
```

Comment:
can't we use `ethers` for this since it's already a dependency of the 3 apps using this `common` package?

Reply:
I actually see that the viem implementation it's the following, so it would be better to copy it inside instead of including another big package like this:

```
const addressRegex = /^0x[a-fA-F0-9]{40}$/
export function isAddress(address: string): address is Address {
  return addressRegex.test(address)
}

```

Reply:
viem is treeshakable, so in this case, we are only importing this exact function, and the impact on the budle size would therefore be exactly the same.

Reply:
ethers is fine but it's not threeshakeable as far as I am aware, so it's better to include viem since it's tiny rather than potentially duplicating another huge ethers version into our bundle.

Reply:
We should just use ethers.utils.isAddress